# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1367205703 14400
#      Sun Apr 28 23:21:43 2013 -0400
# Node ID e74ef19d2268b1a07ac8d504266f8086a0b59550
# Parent  868d0bcfabaebebebfc8a0c8bda66fb64368c15e
use octave_idx_type instead of int

* ls-mat-ascii.cc (looks_like_mat_ascii_file): USe octave_idx_type
instead of int for temporary variables.

diff --git a/libinterp/interp-core/ls-mat-ascii.cc b/libinterp/interp-core/ls-mat-ascii.cc
--- a/libinterp/interp-core/ls-mat-ascii.cc
+++ b/libinterp/interp-core/ls-mat-ascii.cc
@@ -413,18 +413,18 @@ bool
 looks_like_mat_ascii_file (const std::string& filename)
 {
   bool retval = false;
 
   std::ifstream is (filename.c_str ());
 
   if (is)
     {
-      int nr = 0;
-      int nc = 0;
+      octave_idx_type nr = 0;
+      octave_idx_type nc = 0;
 
       get_lines_and_columns (is, nr, nc, filename, true, true);
 
       retval = (nr != 0 && nc != 0);
     }
 
   return retval;
 }
